waspc/cli/Command/Compile.hs

```diff
@@ -0,0 +1,35 @@
+module Command.Compile
+    ( compile
+    ) where
+
+import           Control.Monad.IO.Class (liftIO)
+import           Data.List              (find, isSuffixOf)
+import qualified Path                   as P
+
+import qualified Common
+import           CompileOptions         (CompileOptions (..))
+import qualified Lib
+import           StrongPath             (Abs, Dir, Path, (</>))
+import qualified StrongPath             as SP
+import qualified Util.IO
+
+
+-- | Compiles Wasp source code in waspRoot directory and generates a project in given outDir directory.
+compile :: Path Abs (Dir Common.WaspProjectDir) -> Path Abs (Dir Lib.ProjectRootDir) -> IO (Either String ())
+compile waspRoot outDir = do
+    maybeWaspFile <- findWaspFile waspRoot
+    case maybeWaspFile of
+        Nothing -> return $ Left "No .wasp file present in the root of Wasp project."
```

Comment:
shouldn't this be `.wasp dir`?

Reply:
or `.wasproot` file?

Reply:
aha no, this is e.g. main.wasp, now I get it. Maybe now it somehow differently, e.g. `*.wasp` or sth like that, otherwise it sounds like there should be `.wasp` file.

Reply:
You are right it is confusing! Now it is `*.wasp` file as you suggested.